### PR TITLE
feat(SphereHandleRepresentation): Expose property and selectProperty

### PR DESCRIPTION
**Pull request by finetjul**
The selection color of a handle widget must be editable

### CODE REVIEW

File: Sources/Interaction/Widgets/SphereHandleRepresentation/index.js [hunk 1]
```diff
@@ -350,7 +350,11 @@ export function extend(publicAPI, model, initialValues = {}) {
   // Inheritance
   vtkHandleRepresentation.extend(publicAPI, model, initialValues);
 
-  macro.setGet(publicAPI, model, ['translationMode']);
+  macro.setGet(publicAPI, model, [
+    'translationMode',
+    'property',
```
Comment (jourdain) [inline Sources/Interaction/Widgets/SphereHandleRepresentation/index.js]: Is there a `property` in the model?

Comment (finetjul) [reply Sources/Interaction/Widgets/SphereHandleRepresentation/index.js]: Yes, line 338

Comment (jourdain) [reply Sources/Interaction/Widgets/SphereHandleRepresentation/index.js]: I don't see it...

Comment (laurennlam) [reply Sources/Interaction/Widgets/SphereHandleRepresentation/index.js]: @jourdain , the property of SphereHandleRepresentation is defined here : https://github.com/finetjul/vtk-js/blob/e706894165f7afbd45fc57978b902226ba2c92ea/Sources/Interaction/Widgets/SphereHandleRepresentation/index.js#L338

**Pull request merged**